tools/cuttlefish-host-image-installer/kernel_build_deb.sh

```diff
@@ -0,0 +1,2258 @@
+#!/bin/sh
+
+set +e
+
+SELFPID=$$
+renice 10 -p "$SELFPID"
+ionice -c 3 -p "$SELFPID"
+
+set -e
+
+if [ x"${CI_PIPELINE_ID}" = x ]; then
+    export CI_PIPELINE_ID=1
+fi
+
+TDIR=kernel-build-space/buildresult
+SDIR=kernel-build-space/source/kernel
+
+rm -rf ${TDIR}
+
+mkdir -p ${TDIR}
+
+# Make a copy of AOSP's kernel. deb-pkg target will make the source tree
+# unclean. So we make a copy of it and then build inside the copy.
+cp -R ${SDIR}/common ${TDIR}/common
+cp -R ${SDIR}/common-modules ${TDIR}/common-modules
+cp -R ${SDIR}/.repo ${TDIR}/.repo
+
+# Build the kernel
+cd ${TDIR}/common
+export ARCH=arm64
+export CROSS_COMPILE=aarch64-linux-gnu-
+KERVER=$(make kernelversion)
+make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- defconfig -j32
+for i in arch/arm64/configs/gki_defconfig ../common-modules/virtual-device/virtual_device_core.fragment ../common-modules/virtual-device/linux_distro.fragment; do 
+    ./scripts/kconfig/merge_config.sh -O . .config ${i}
+done
+cat <<EOF > ../extraconfig
```

Comment:
nit: rather than a shell script that writes a long file and then uses it somewhere, can the long file exist on its own and can the shell script reference it?

Reply:
This is more kernel related work and is done by Paul. I have to discuss with Paul and see if we could have a long file existing and have the script reference it.

But I don't think we could do it in this CL.

Reply:
I don't see what's controversial about this comment. Right above this `cat > ../extraconfig` statement there's a `for` loop that invokes `./scripts/kconfig/merge_config.sh` with a few files, and right after there's a `./scripts/kconfig/merge_config.sh` invocation referencing `../extraconfig`.

So what makes `../extraconfig` different that it has to be written from this shell script rather than committed as its own file?